.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: hashicorp/terraform:latest
```

Comment:
I decided to use `latest` here because it would (1) cut down on maintenance, and (2) give us a heads up when things become deprecated upstream.